.github/workflows/srt.yml

```diff
@@ -173,7 +173,7 @@ jobs:
           export NETCDF=$HOME/netcdf-fortran
           export PATH=$NETCDF/bin:$PATH
           export LD_LIBRARY_PATH=$NETCDF/lib:$HOME/pnetcdf/lib:$LD_LIBRARY_PATH
-          python -m pip install pytest pytest-cov
+          python -m pip install pytest pytest-cov six
```

Comment:
We should no longer need six as we no longer support python 2.

Reply:
> Let's just remove the six package completely - you've already started doing it.

Sounds good, I'll remove and note it in the PR description.